includes/library/zencart/DashboardWidget/src/Logs.php

```diff
@@ -20,6 +20,12 @@
  */
 class Logs extends AbstractWidget
 {
+  public function __construct($widgetKey, $widgetInfo = NULL) 
+  {
+      parent::__construct($widgetKey, $widgetInfo);
+      $count = 0;
```

Comment:
Shouldn't `$count` be `$this->count` ?

And a declaration of the variable as `protected $count;` above the constructor?